index.html

```diff
@@ -809,6 +809,21 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
 =&gt; [[1, 3, 5], [0, 2, 4]]
 </pre>
 
+      <p id="isEqual">
+        <b class="header">isEqual</b><code>_.isEqual(list, other)</code>
+        <br />
+        Performs an optimized deep comparison between one <b>list</b> and the <b>other</b>, to determine
```

Comment:
I can see what you mean, it seems that it has some undocumented behaviors. It's a tough one. I pushed the change in the first place because it had taken me some time today to track `isEqual` down, since the Collections-Arrays-Objects distinction suggests that anything under *Objects* will not work on arrays, and vice versa.

Ultimately, *Utility* seems to make the most sense. What would you suggest instead of **list** (that is not **object** to avoid the above ambiguity) as a general term for all the supported types? Maybe **value**?